packages/core/src/codewhispererChat/controllers/chat/messenger/messenger.ts

```diff
@@ -334,6 +342,24 @@ export class Messenger {
             })
     }
 
+    private extractCodeBlockLanguage(message: string): string {
```

Comment:
seems like something that should be tested. and maybe Q team wants to define a `shared/markdown.ts` file (i don't know how much other things like this are in this or similar modules)

Reply:
Addressed this